db/migrate/20160331111827_add_stat_containerimage_create_rate_to_metric_rollups.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatContainerimageCreateRateToMetricRollups < ActiveRecord::Migration[5.0]
+  def change
+    add_column :metric_rollups, :stat_containerimage_new_images_rate, :integer
```

Comment:
@Fryguy 
> How is this stat different from create_rate, btw?

Create implies something from nothing while in our case we simply register in our system somthing that already exist, btw new-images is also problematic from the same reason :-(

A better wording is very needed here.